test/integration/builds_int_test.go

```diff
@@ -150,8 +167,18 @@ func (suite *BuildsIntegrationTestSuite) TestBuilds_Download() {
 	)
 	cp.ExpectExitCode(0, e2e.RuntimeSourcingTimeoutOpt)
 
+	var buildID string
+	if runtime.GOOS == "windows" {
+		// On Windows we need the specific build ID as the terminal buffer is not
+		// large enough to display all the builds
+		buildID = "dbf05bf8-4b2e-5560-a329-b5b70bc7b0fa"
```

Comment:
Unfortunate work around that means we may have to periodically update this test. If there is a way to do this that I'm not seeing then please let me know. Otherwise, I will file a follow up story to increase the output buffer on Windows.

Reply:
Can we `cp.Expect()` "bzip2@1.0.8", `cp.ExpectRe()` a UUID, extract that UUID from the output, and then use it? Something like https://github.com/ActiveState/cli/blame/master/cmd/state-svc/test/integration/svc_int_test.go#L56-L67.

I feel like marshaling stdout to JSON is brittle. The output format could change, there could be sporadic I/O issues on CI, etc.

Reply:
I prefer the JSON approach here as this test will no fail if the output format changes, which is something we should be looking out for. The sporadic I/O issues would also affect the regex approach.